forge-gui/res/cardsfolder/upcoming/redemptor_dreadnought.txt

```diff
@@ -0,0 +1,13 @@
+Name:Redemptor Dreadnought
+ManaCost:5
+Types:Artifact Creature Astartes Dreadnought
+PT:4/4
+K:Trample
+S:Mode$ RaiseCost | ValidCard$ Card.Self | Type$ Spell | Cost$ ExileFromGrave<Y/Creature/creature card> | EffectZone$ All | ImprintExiled$ True | Description$ Fallen Warrior — As an additional cost to cast this spell, you may exile a creature card from your graveyard.
```

Comment:
This seems wrong still

This might need engine work... other cards like this have always been "you may exile any number" or "you may sacrifice any number"

Reply:
Yeah tried it like Gorex, the Tombshell. Will probably just remove this card for now

Reply:
that might work if you combine it with `XMaxLimit$ 1`

Reply:
Not having much succes with testing. Might need engine work like you said.